browserdec.py : visuals + added a note-parser + added scan for deleted logins

Banaanhangwagen opened the pull request:
v2023.06 - updated diana-browserdec.py - tested on v117 (canary) and v114 (stable)
* 🌈: Did some visual updates
* 📝: Added the decryption of "Notes" that a user optionally can add when saving a password.
* 🥷: Also added a quick scan for deleted logins.

* Example
  ![image](https://github.com/Banaanhangwagen/diana/assets/25983612/49c900e9-c8ac-4bbd-b795-5f0a39b88ef2)

---

Banaanhangwagen committed:
Update diana-browserdec.py

Did some visual updates.

Added the decryption of "Notes" that a user optionally can add when saving a password.

Also added a quick scan for deleted logins.

Banaanhangwagen committed:
Update requirements.txt

Banaanhangwagen committed:
Update README.md

Banaanhangwagen committed:
Update README.md

tijldeneut commented:
Nice one, testing it now.

tijldeneut commented:
Hi, there's a bug/exception in line 393 where "lstNotes" is undefined if the user does not provide a login file.
Also the encoding header is specified twice (line 1 and line 29).

Since this is a different coding style and graphical view than the other scripts, can I rename it to diana-browserdec-ng.py or something in that trend? I would also add your handle to the copyright message. 

E.g.
```
Copyright 2022, Tijl "Photubias" Deneut <@tijldeneut>
Copyright 2023, Banaanhangwagen <@banaanhangwagen>
```

Thanks

Banaanhangwagen committed:
Update diana-browserdec.py

Banaanhangwagen commented:
Hello,
i made the changes as suggested:
- fixed the exception when no login data file is provided
- deleted the second encoding header
- added my handle

feel free to rename it in your repo to another file
